official/resnet/cifar10_main.py

```diff
@@ -23,7 +23,8 @@
 
 import tensorflow as tf
 
-from official.resnet import resnet
+from official.resnet import resnet_model
+from official.resnet import resnet_util
```

Comment:
resnet_util seems unclear; maybe resnet_run? resnet_loop? I don't think there is a good name for "the stuff that lives around the model," unfortunately.

Reply:
Perhaps it is better to make the support code just resnet.py as before, and the other file something like layer_definitions.py?